template/workitem.update/body.html

```diff
@@ -1,142 +1,150 @@
+              </a>
+            </span>
           </span>
-        </span>
-      </p>
-    </div>
-    <div style="
-          background-color: #f9f9f9;
-          padding: 10px 20px;
-          min-height: 150px;
-          max-width: 700px;
-          text-align: left;">
-      <h3 style="margin-bottom: 5px;">
-        <a href="{{detailURL .webURL .spaceOwner .space .workitem}}" style="
-            color: #0088ce;
-            font-weight: 700;
-            text-decoration: none;">
-          {{index .workitem.Data.Attributes "system.title"}}
+        </p>
+      </div>
+      <div class="content">
+        <h3>
+          <a href="{{detailURL .webURL .spaceOwner .space .workitem}}" class="link">
+            {{index .workitem.Data.Attributes "system.title"}}
+          </a>
+        </h3>
+        <p style="
+            font-size: 16px;">
+          <table>
+            <tr style="height: 30px;">
+              <td style="width: 100px; text-align: right;">
+                <span style="font-weight: 700;">Status</span>
+              </td>
+              <td style="text-align: left; padding-left: 10px; max-width: 700px;">
+                <span style="color: #3f9c35;">{{index .workitem.Data.Attributes "system.state"}}</span>
+              </td>
+            </tr>
+          </table>
+        </p>
+      </div>
+      <div class="footer">
+        <a href="{{detailURL .webURL .spaceOwner .space .workitem}}" class="link-footer">
+          Link to this item
         </a>
-      </h3>
-      <p style="
-          font-size: 16px;">
-        <table>
-          <tr style="height: 30px;">
-            <td style="width: 100px; text-align: right;">
-              <span style="font-weight: 700;">Status</span>
-            </td>
-            <td style="text-align: left; padding-left: 10px; max-width: 700px;">
-              <span style="color: #3f9c35;">{{index .workitem.Data.Attributes "system.state"}}</span>
-            </td>
-          </tr>
-        </table>
-      </p>
-    </div>
-    <div style="
-            margin-top: 25px;
-            margin-bottom: 5px;
-            display: inline-flex;">
-      <span style="width: 150px;">
-        <a href="{{detailURL .webURL .spaceOwner .space .workitem}}"
-            style="
-              color: #0088ce;
-              text-decoration: none;
-              font-weight: 500;
-              vertical-align: sub;">
-              Link to this item</a>
-      </span>
-      <span style="width: 150px;">
-        <a href="{{detailURL .webURL .spaceOwner .space .workitem}}"
-            style="
-              color: #0088ce;
-              text-decoration: none;
-              font-weight: 500;
-              vertical-align: sub;">
-              Comment</a>
-      </span>
+        <span class="divider-vertical"></span>
+        <a href="{{detailURL .webURL .spaceOwner .space .workitem}}" class="link-footer">
```

Comment:
correct me if I am wrong, but it's hard to know what has been updated in the workitem?

Reply:
https://github.com/openshiftio/openshift.io/issues/1990